scripts/test_screeps_tencent_batch_rl_runner.py

```
@@ -4302,6 +4302,138 @@ def capture_run_cp(
+            args = controller_args()
+            args.known_hosts_path = str(Path(temp_dir) / "known_hosts")
+            known_hosts = Path(args.known_hosts_path)
+            known_hosts.write_text(stale_key + "\n", encoding="utf-8")
+            controller = runner.Controller(args=args, run_id="run-test", artifact_dir=Path(temp_dir))
+            controller.public_ip = "203.0.113.10"
+            controller.known_hosts_prepared_public_ips.add("203.0.113.10")
+            keyscan_calls = 0
+            operation_commands: list[list[str]] = []
+
+            def fake_run_cp(
+                name: str,
```

Ruff (0.15.13)

[warning] Unused function argument: `name`

(ARG001)